content/angle-grinder.md

```diff
@@ -0,0 +1,30 @@
+# Angle Grinder
+
+Angle grinders are versatile tools that are also extremely dangerous. This grinder can take 4 1/2” diameter discs of different types and spins them at very high RPM. Discs you can use include abrasive cut-off wheels, flap discs (course sandpaper), and other more technical heads such as wood power carvers, wire wheels, and more..
+
+## ⚠️ Safety:
+
+- Wear safety glasses and hearing protection and a full face shield.
+- Use a mask or respirator for breathing protection whenever grinding that creates metal or wood dust.
+- Do not wear jewelry or clothing with ties or other loose dangly bits. Tie up long hair.
+- Wear the appropriate gloves to protect against sparks or wood debris.
```

Comment:
It's a rotary tool so many some glove caution would also be good.

Reply:
I've edited a bit, though OSHA still recommends gloves for grinder use.